tests/JobDefinitionParserTest.php

```diff
@@ -240,37 +264,17 @@ public function testCreateJobDefinitionWithConfigIdAndBackend(
 
     public function testCreateJobDefinitionWithConfigNotFound(): void
     {
-        $configuration = [
-            'id' => 'my-config',
-            'version' => '1',
-            'state' => [],
-            'rows' => [],
-            'configuration' => [
-                'runtime' => [
-                    'foo' => 'bar',
-                ],
-            ],
-        ];
-
-        $jobData = [
+        $job = $this->createJob([
             'status' => JobInterface::STATUS_CREATED,
             'runId' => '1234',
             'projectId' => 'my-project',
             'componentId' => 'my-component',
             'configId' => 'my-config',
             'branchType' => null,
-        ];
-
-        $encryptor = $this->createMock(ObjectEncryptor::class);
-        $encryptor->method('decryptForConfiguration')->with($configuration)->willReturn($configuration);
+        ]);
 
-        $job = new Job(
-            new JobObjectEncryptor($encryptor),
-            $this->createMock(StorageClientPlainFactory::class),
-            $jobData
-        );
-        $storageApiClient = $this->createMock(Client::class);
-        $storageApiClient->method('apiGet')
+        $parserStorageApiClient = $this->createMock(Client::class);
+        $parserStorageApiClient->method('apiGet')
```

Comment:
Btw, myslím jsi byl zrovna pryč, co si myslíš o tomhle https://keboolaglobal.slack.com/archives/C0553EM2RGX/p1686903052846019 vždycky když vidím namockovanej ten apiGet, tak ve mě něco umře (a jo vidím, že to tam bylo i předtím :) )

Reply:
:smile: parkrat me to sedelo v zaludku, ale pak jsem na to (stejne jako na vsechno okolo DX SAPI klienta) rezignoval a naucil se to mechanicky delat porad stejne.